lib/api/controllers/readController.js

```diff
@@ -25,32 +25,11 @@ module.exports = function ReadController (kuzzle) {
     return kuzzle.services.list.readEngine.count(requestObject);
   };
 
-  this.listCollections = function (requestObject, context) {
+  this.listCollections = function (requestObject) {
```

Comment:
why did you remove filterCollections() method ?
We need to list  only collections that the user is allowed to read, that was the purpose of this method (that probably should be fixed to follow your changes).

I don't see how this kind of filter is implemented in your PR.